linearElasticity/curvedCantilever/coupledUnsLinearGeometryLinearElastic/Allrun

```diff
@@ -7,7 +7,7 @@
 source solids4FoamScripts.sh
 
 # Compile functionObject for generation of analytical solution
-compileLibrary analyticalCurvedCantilever
+compileLibrary curvedCantileverAnalyticalSolution
```

Comment:
I did not know about the `compileLibrary` function; that is convenient. Unfortunately, it seems to only be in foam extend.

One option is that we could add this function to solids4FoamScripts.sh. For now, we don't need to do anything.

Reply:
For now, I'm adding cases using `fe41`, and will check/modify them using other versions later. This function object is already in the development branch, so `solids4Foam::rcompileLibrary` is not needed; I guess that later we can easily add it if required.